online-bookstore/online-bookstore/src/main/java/com/example/online_bookstore/implementation/BookOrderServiceRepo.java

```diff
@@ -0,0 +1,67 @@
+package com.example.online_bookstore.implementation;
+
+import com.example.online_bookstore.models.BookOrder;
+import com.example.online_bookstore.repository.BookOrderRepo;
+import com.example.online_bookstore.service.BookOrderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+public class BookOrderServiceRepo implements BookOrderService {
+
+    @Autowired
+    BookOrderRepo repo;
```

Comment:
Instead of using @Autowired use @AllaArgsConstructor at the top of the class, look into why this is a better practice.